rasa/tts.py

```diff
@@ -0,0 +1,20 @@
+import requests
```

Comment:
Is this file still needed?

Reply:
For future debugging purposes it might be useful.